docs/quickstart.md

```diff
@@ -124,6 +124,12 @@ It takes the following parameters:
 * `--pip-legacy-resolver` - [optional] Use old pip dependency resolver by adding flag '--use-deprecated=legacy-resolver'
     to pip install command. The default is`False`. NOTE: This flag is deprecated and will be removed from pip in the future.
     Instead of using this flag, the correct solution would be to fix the packages your project depends on to work properly with the new resolver. Additionally, this flag is not compatible with pip version `23.2`. Use `23.2.1` instead.
+* `--pip-custom-flag` - [optional] Additional flag(s) that will be added to the `pip install` command.
```

Comment:
If you run

```
poetry run ucc-gen build --source tests/testdata/test_addons/package_global_config_everything/package --pip-custom-flag="--no-compile --prefer-binary --ignore-installed --report report.json" --pip-custom-flag="--progress-bar on"
```

it takes only last `--pip-custom-flag`.

```
INFO: Executing: python3 -m pip install -r "tests/testdata/test_addons/package_global_config_everything/package/lib/requirements.txt" --target "/Users/arys/artemrys/addonfactory-ucc-generator/output/Splunk_TA_UCCExample/lib" --progress-bar on
```

Reply:
isn;t that a case for every flag? e.g. 
`ucc-gen build --source /Users/sgoral/UCC_TEST/demo_addons/demo_addon_release/package --source asd `
`NotADirectoryError: /Users/sgoral/Splunk_repos/addonfactory-ucc-generator/asd not found.`

Reply:
hm, I do not think that's expected in general.

let's tackle this in a separate PR.